h5bp/location/expires.conf

```diff
@@ -17,21 +17,26 @@ location ~* \.(?:manifest|appcache|html?|xml|json)$ {
 # Feed
 location ~* \.(?:rss|atom)$ {
   expires 1h;
-  add_header Cache-Control "public";
 }
 
 # Media: images, icons, video, audio, HTC
-location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|htc)$ {
+location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|mp4|ogg|ogv|webm|htc)$ {
   expires 1M;
   access_log off;
-  add_header Cache-Control "public";
+}
+
+# Media: svgz files are already compressed.
+location ~* \.svgz$ {
+  expires 1M;
+  access_log off;
+  gzip off;
```

Comment:
The other types are not in the `gzip_types` list. SVG/SVGZ is in that list.
Perhaps the comment above the location block needs to be changed to something like:

```
# We need to handle svgz files separately from svg files.
# Since they have the same MIME type, Nginx will try to apply the gzip settings even
# though svgz files are already compressed.
```